apps/testingaccessibility/src/components/sale-banner.tsx

```diff
@@ -41,6 +41,12 @@ const CountdownRenderer: React.FC<React.PropsWithChildren<CountdownProps>> = ({
     minutes,
     seconds,
   })
+  const [isMounted, setIsMounted] = React.useState(false)
+  React.useEffect(() => {
+    setIsMounted(true)
+  }, [])
+
+  if (!isMounted) return null
```

Comment:
this is unrelated but wanted to fix this error that the sale banner has been throwing